x-pack/docs/en/security/authorization/mapping-roles.asciidoc

```diff
@@ -2,17 +2,34 @@
 [[mapping-roles]]
 === Mapping users and groups to roles
 
-If you authenticate users with the `native` or `file` realms, you can manage
-role assignment by using the <<managing-native-users,user management APIs>> or
-the <<users-command,users>> command-line tool respectively.
+Role mapping is supported by all realms except `native` and `file`.
 
-For other types of realms, you must create _role-mappings_ that define which
-roles should be assigned to each user based on their username, groups, or
-other metadata.
+TIP:  Those two realms assign roles directly to users, they do not support role mapping.
+Native realms use <<managing-native-users,user management APIs>>.
+File realms use <<roles-management-file,File-based role management>>.
```

Comment:
I'd make this part of the first paragraph, rather than a tip.

```suggestion
The native and file realms assign roles directly to users.
Native realms use <<managing-native-users,user management APIs>>.
File realms use <<roles-management-file,File-based role management>>.
```